rule-template/cuevox.yaml

```diff
@@ -1,4 +1,4 @@
-uid: jm:cuevox
+uid: 'jm:cuevox'
```

Comment:
Have you manually adjusted the YAML or used the script I've provided: https://github.com/JanMattner/voice-control-openhab/blob/16134f384182a65f5ed9a4d9dd934a2d3f8a8155/package.json#L12

The workflow uses that and reports a difference.

Please use "npm run update-yaml" and check it in.

Reply:
Strange... somehow the script behaves differently on different systems. I think I will change the way how the YAML is updated... Anyway, that should not block this PR.